docs/tempo/website/grafana-agent/_index.md

```diff
@@ -0,0 +1,126 @@
+
+#### Attaching metadata with Prometheus Service Discovery
+
+Prometheus Service Discovery mechanisms enable you to attach the same metadata to your traces as your metrics.
+For example, for Kubernetes users this means that you can dynamically attach metadata for namespace, pod, and container name of the container sending spans.
+
+```
+tempo:
+  ...
+  scrape_configs:
+  - bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+    job_name: kubernetes-pods
+    kubernetes_sd_configs:
+    - role: pod
+    relabel_configs:
+    - source_labels: [__meta_kubernetes_namespace]
+      target_label: namespace
+    - source_labels: [__meta_kubernetes_pod_name]
+      target_label: pod
+    - source_labels: [__meta_kubernetes_pod_container_name]
+      target_label: container
+    tls_config:
+      ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+      insecure_skip_verify: false
+```
+
+This feature isn’t just useful for Kubernetes users, however.
+All of Prometheus' [various service discovery mechanisms](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#configuration-file) are supported here.
+This means you can use the same scrape_configs between your metrics, logs, and traces to get the same set of labels,
+and easily transition between your observability data when moving from your metrics, logs, and traces.
+
+To configure it, refer to the `scrape_configs` block in the [config reference](https://github.com/grafana/agent/blob/main/docs/configuration-reference.md#tempo_instance_config).
+
+#### Trace discovery through automatic logging
+
+Automatic logging provides an easy and fast way of getting trace discovery through logs.
+Automatic logging writes a well formatted log line to a Loki instance for each span that is processed in the pipeline.
+
+For a closer look into the feature, visit [Automatic logging](./automatic-logging.md).
+
+#### Tail-based sampling
+
+The Agent implement tail-based sampling for distributed tracing systems and multi-instance Agent deployments.
+With this feature, sampling decisions can be made based on data from a trace, rather than exclusively with probabilistic methods.
+
+For a detailed description, go to [Tail-based sampling](./tail-based-sampling.md).
+
+#### Generating metrics from spans
+
+The Agent can take advantage of the span data flowing through the pipeline to generate Prometheus metrics.
+
+Go to [Span metrics](./span-metrics.md) for a more detailed explanation of the feature.
+
+### Exporting spans
+
+The Grafana Agent can export traces to multiple different backends for every tracing pipeline.
+Exporting is built using OpenTelemetry Collector's [OTLP exporter](https://github.com/open-telemetry/opentelemetry-collector/blob/846b971758c92b833a9efaf742ec5b3e2fbd0c89/exporter/otlpexporter/README.md).
+The Agent supports exporting tracing in OTLP format.
```

Comment:
We already said this. Probably we can remove this from the intro.

Reply:
Fixed, thanks!